themes/delphi/layouts/partials/nav.html

```diff
@@ -31,15 +31,14 @@
   </div>
 </nav>
 
-
 <!-- Mobile nav -->
 <nav class="uk-hidden@s mobile-nav">
-  {{partial "delphi-text-logo.html"}}
+  {{partial "delphi-text-logo.html" . }}
   <a class="uk-navbar-toggle" uk-navbar-toggle-icon href=""></a>
     <div uk-dropdown="mode: click" class="dropdown-mobile">
       <ul class="uk-nav uk-navbar-dropdown-nav">
         {{ range .Site.Menus.main.ByWeight }}
-        <li class="{{if .HasChildren }}menu-parent{{end}} menu-element">
+        <li class="{{if .HasChildren }}menu-parent{{end}} menu-element {{ if (eq $currentPage.Section .Identifier) }}nav-active{{ end }}">
```

Comment:
I'm not sure if @Letsgohyde meant to have the active indicator in the mobile menu.

Reply:
I remove it